src/drivers/CMakeLists.txt

```diff
@@ -4,5 +4,8 @@ if(CONFIG_IMX8)
 	add_subdirectory(imx)
 else()
 	add_subdirectory(intel)
+endif()
+
+if(CONFIG_DW_DMA)
```

Comment:
What with the gpio.c and ssi-spi.c in the same directory?
It doesn't feel right to make them dependent on `CONFIG_DW_DMA`.

Reply:
@tlauda TBH I don't know why they are there at all, it's some suecreek specific stuff. If it's also from DesignWare then we need CONFIG_DW symbol, if they are not, then probably they shouldn't be there.
 
@lyakh @lgirdwood  Why are these files there?

Reply:
@Jiangxinx  @tlauda these files are both DW IP.